rules/auth0_rules/auth0_mfa_policy_disabled.py

```diff
@@ -10,7 +10,7 @@ def rule(event):
     request_path = deep_get(
         event, "data", "details", "request", "path", default="<NO_REQUEST_PATH_FOUND>"
     )
-    request_body = deep_get(event, "data", "details", "request", "body", default=[-1])
```

Comment:
I updated this default value since the sample log(s) either returned a dictionary or `body: []`, so `-1` didn't seem to make sense (and it failed the test when I ran it against the new `deep_get` code).